src/main.tf

```diff
@@ -47,17 +37,3 @@ resource "azurerm_storage_account" "main" {
     }
   }
 }
-
-resource "azurerm_private_endpoint" "main" {
```

Comment:
This looks like it's used for on-prem access to Azure. Since it's here I was thinking this would magically allow access from the subnet to storage but that does not appear to be the case.

Reply:
This was about a 15 hour distraction.